packages/lib/constants/feature-flags.ts

```diff
@@ -23,6 +23,7 @@ export const FEATURE_FLAG_POLL_INTERVAL = 30000;
 export const LOCAL_FEATURE_FLAGS: Record<string, boolean> = {
   app_billing: NEXT_PUBLIC_FEATURE_BILLING_ENABLED() === 'true',
   app_document_page_view_history_sheet: false,
+  app_passkey: WEBAPP_BASE_URL === 'http://localhost:3000', // Temp feature flag.
```

Comment:
👀

Reply:
Passkeys would default to false when feature flags are not enabled

So to allow local development I just left it as such until we gain confidence so set the default as true

Reply:
Fair enough 😄